driftbase/api/players/__init__.py

```diff
@@ -80,18 +86,28 @@ def drift_init_extension(app, **kwargs):
 
 def _handle_set_player_name_from_seasons(*args, **kwargs):
     # C/P from _patch method below to quickly support letting seasons change player names
+
     new_name = kwargs.get("player_name")
     player_id = kwargs.get("player_id")
     log.info(f"Handling event from drift-seasons: {args=} - {kwargs=}")
-    my_player = g.db.query(CorePlayer).get(player_id)
+    db_session = _get_db()
+    my_player = db_session.query(CorePlayer).get(player_id)
     if not my_player:
         log.warning(f"Player {player_id} does not exist")
         return
 
     old_name = my_player.player_name
     my_player.player_name = new_name
-    g.db.commit()
+    db_session.commit()
     log.info("Player changed name from '%s' to '%s'", old_name, new_name)
+
+    message_data = dict(
+        player_id=player_id,
+        player_name=my_player.player_name,
+        player_uuid=my_player.player_uuid.hex if my_player.player_uuid else None,
+    )
+
+    _get_shoutout().message("player_name_updated", **message_data)
```

Comment:
Since we now support custom topic names, I think we should use `player:name_updated` here and in the listener.